src/azure-cli/azure/cli/command_modules/resource/_help.py

```diff
@@ -2141,3 +2175,139 @@
     text: >
         az tag update --resource-id /subscriptions/{sub-id}/resourcegroups/{rg}/providers/Microsoft.Compute/virtualMachines/{vmName} --operation delete --tags key1=value1
 """
+
+helps['template-specs'] = """
+type: group
+short-summary: Manage template specs at subscription or resource group scope.
+"""
+
+helps['template-specs create'] = """
+type: command
+short-summary: Create or update a template spec and or template spec version.
+parameters:
+  - name: --resource-group -g
+    type: string
+    short-summary: Name of resource group.
+  - name: --name -n
+    type: string
+    short-summary: Name of the template spec.
+  - name: --version -v
+    type: string
+    short-summary: The template spec version.
+  - name: --template-file -f
+    type: template_spec
+    short-summary: The path to the template file.
+  - name: --location -l
+    short-summary: The location to store the template-spec.
+  - name: --display-name -d
+    type: string
+    short-summary: The display name of the template spec.
+  - name : --description
+    type: string
+    short-summary: The description of the root template spec.
+  - name : --vdescription
```

Comment:
@zhoxing-ms is it okay if we use acronym?

Reply:
Good question. It's better not to abbreviate so that the meaning of parameter name will be clearer.
Could you change it to a short, non-abbreviated parameter name?

Reply:
Thanks, that's good to know. @detienne20, let's change it to `--version-description`.